### ISSUE TITLE
Wireframing & Layout: Due March 15

### ISSUE THREAD

**Issue by meganspaulding** [MeganSpauldingCreative/GWG-animal-card-generator#3]
We've officially started! YAY! We are waiting on one other sign up who has committed on slack, but it looks like I am the only one on the Wireframing team, which will be our first step

We will get started on Wireframing and Layout today, which will give a base to our project. I will have first rounds submitted tonight for us to make notes and suggestions on! Let's plan on having a layout locked down by the 15th!

I have updated the Read Me file with a tentative timeline for this project, so be sure to check that out!

**Comment by dimakm** [comment]
Hi Megan, I read the Readme file.
So lets start planning, first, we need a form in the Html page where we can insert the information, then when we submit the form a card will show up, right?

**Comment by dimakm** [comment]
I started with a very basic form, without any formatting or CSS. 
do you think a good Idea having the animal group like Mammals Reptiles and so, as a radio button group so we can choose one choice when entering the animal info !
here's the codepen I created for the form, very basic with no formatting:

https://codepen.io/Dima-KM/pen/aYdYdo

**Comment by meganspaulding** [comment]
Perfect! I will take a look at the code when the kids go to sleep! I’ve committed it for now for others to be able to take a look at!

**Comment by meganspaulding** [comment]
I have also added a wideframe file! Please check this out for images with base layouts made. 

Other than “add”, “delete”, and “edit”, are there any other features you think we should have?

**Comment by stealing-pencils** [comment]
These look fantastic already!  I think the form Dima, is great and the wireframe files already look brilliant Megan.  I was wondering, whether it's worth laying out the objective of these cards i.e. if they're meant to be educational or entertaining etc?  Who are they meant to be for?  I only ask because that may change the way we present them.  If they're for kids, then they can be presented in a simpler / cute style, whereas if they're more for entertainment purposes, they could be more jokey with an 'Exploding Kitties' type look.  

Separately, Megan when you included the 'add', 'delete' and 'edit' features, were they to provide the user with the means of 'adding' another card, 'deleting' a card from their collection and finally 'editing' a card?  I wasn't quite sure what they were going to edit.  I totally get the add and delete buttons, but just wanted to make sure I was on the same page with edit.  Thanks!

**Comment by YOURsammich** [comment]
Do you think we should show the edit button on the cards? Like a edit button will appear on the top right corner of a card when you hover over it or a little pen icon.

That way you don't have to click edit and than click on a card.

**Comment by YOURsammich** [comment]
I was thinking that maybe we could have the "delete" and "edit" options on the card.
Like when you hover over a card a pen and trashcan icon could appear in the top right corner.
This way you don't have to click edit or delete than click on the card you want to change, you just press one button.

we can use https://material.io/icons/ for the icons

If we do it this way I know the top bar that has the add, edit and delete buttons would look kind of weird only having 1 button so maybe we can also put how many total cards we have on that bar.

**Comment by meganspaulding** [comment]
@YOURsammich that is a great idea! What if we removed that line completely and added a "+" card, like attached. What do you think @dimakm @stealing-pencils?
![animal card wireframe_2nd-02](https://user-images.githubusercontent.com/35813751/37571573-669d4670-2acc-11e8-8cad-0feeb7fc05c1.png)

**Comment by YOURsammich** [comment]
@MeganSpauldingCreative Nice, that looks much better

**Comment by stealing-pencils** [comment]
Yeh I think that's a great idea @YOURsammich and @MeganSpauldingCreative, that looks great!  The pen and trashcan hover icons would be great and the + card, would look really round the page off nicely.
